src/frontends/onnx/frontend/src/op/blackmanwindow.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "openvino/core/deprecated.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
+
+#include "ngraph/node.hpp"
+#include "onnx_import/core/node.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_17 {
```

Comment:
Sorry, looks like I was misleading you. Because you are adding new operations, they should be in set_1, you did it right initially.

Reply:
No problem 😃